pytmc/bin/pragmalint.py

```diff
@@ -84,6 +87,35 @@ def lint_pragma(pragma):
     match = PRAGMA_RE.match(pragma)
     if not match:
         raise LinterError()
+
+    try:
+        pragma_setting = match.groupdict()["setting"]
+    except KeyError:
+        # if no configuration region in the pragma was detected then fail
+        raise LinterError()
+
+    config_lines = PRAGMA_LINE_RE.findall(pragma_setting)
+    if len(config_lines) == 0:
+        # It is not acceptable to lack configuration lines
+        # At a minimum "pv: " must exist
+        raise LinterError()
+
+    config_lines_detected = 0
+    pv_line_detected = 0
+
+    for line in config_lines:
+        line_match = PRAGMA_SETTING_RE.match(line)
+        if line_match:
+            config_lines_detected += 1
+            pv_match = PRAGMA_PV_LINE_RE.search(line)
+            if pv_match:
+                pv_line_detected += 1
+
+    # There shall be at one config line at minimum
+    # There shall be a config line for pv even if it's just "pv:"
+    if (config_lines_detected <= 0 or pv_line_detected <= 0):
```

Comment:
Let's break this up into 2 cases, raising the exception with the applicable message from above.